lib/matplotlib/tests/test_axes.py

```diff
@@ -5230,6 +5230,24 @@ def test_pcolor_fast_non_uniform():
     ax.pcolorfast(X, Y, Z.T)
 
 
+def test_pcolor_fast_RGB():
+
+    fig, ax = plt.subplots(1, 1)
+
+    np.random.seed(19680801)
+    C = np.random.rand(10, 10, 3)  # RGB image [0,1]
+    x = np.arange(11, dtype=np.float)
+    y = np.arange(11, dtype=np.float)
+
+    ax.pcolorfast(C)
+    ax.pcolorfast(x, y, C)
```

Comment:
The test on this line and the one just above are now taken care of by test_pcolorfast_colormapped, so they can be removed.

Reply:
Okay, they have been removed https://github.com/matplotlib/matplotlib/pull/8690/commits/af603a93495230161eb3c2354236f986b15769f8